MDANSE_GUI/Src/MDANSE_GUI/InputWidgets/PartialChargeWidget.py

```diff
@@ -155,10 +169,26 @@ class PartialChargeWidget(AtomSelectionWidget):
 
     _push_button_text = "Partial charge helper"
     _default_value = "{}"
-    _tooltip_text = "Specify the partial charges that will be used in the analysis. The input is a JSON string, and can be created using the helper dialog."
+    _tooltip_text = (
+        "Specify the partial charges that will be used in the analysis."
+        " The input is a JSON string, and can be created using"
+        " the helper dialog."
+    )
+
+    def __init__(self, *args, _use_list_view=True, **kwargs):
+        """Create the widget for setting atom charges.
+
+        Parameters
+        ----------
+        _use_list_view : bool, optional
+            ignored here. This widget always sets use_list_view=False
 
-    def create_helper(self, traj_data: tuple[str, Trajectory]) -> ChargeHelper:
         """
+        super().__init__(*args, use_list_view=False, **kwargs)
```

Comment:
Again, if you want to solve this, I would put:
```suggestion
        if "use_list_view" in kwargs:
            raise TypeError(f"Cannot use list view with {type(self).__name__}."
        super().__init__(*args, use_list_view=False, **kwargs)
```